File: C++/First Steps in Coding - Basics of C++/Conditional Statements - Advanced/Number in range.cpp
#include <iostream>
#include <cstdlib>
#include <string>
using namespace std;
int main() {
	int num;
	cin >> num;
	switch (num) {
	case 0:
		cout << "No";
		break;
	default:
		if (num <= 100 and num >= -100) {
			cout << "Yes";
		}
		else {
			cout << "No";
		}
	}
}

File: Java/Advanced Conditional Statements/Lab/ACS Lab/src/NumberInRange.java
import java.util.Scanner;

public class NumberInRange {

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        int num = Integer.parseInt(input.nextLine());
        boolean yesNo = false;

        if(num >= -100 && num <= 100){
            if(num != 0){
                yesNo = true;
            }
        }

        if(yesNo){
            System.out.println("Yes");
        } else{
            System.out.println("No");
        }
    }
}
